system/ubloxd/ublox_msg.cc

```diff
@@ -390,7 +390,7 @@ kj::Array<capnp::word> UbloxMsgParser::parse_glonass_ephemeris(ubx_t::rxm_sfrbx_
     eph.setP4(data->p4());
     eph.setSvURA(glonass_URA_lookup.at(data->f_t()));
     if (msg->sv_id() != data->n()) {
-      LOGE("SV_ID != SLOT_NUMBER: %d %d", msg->sv_id(), data->n());
+      LOGE("SV_ID != SLOT_NUMBER: %d %" PRIu64, msg->sv_id(), data->n());
```

Comment:
why not llu?

Reply:
use llu will report error on my laptop.  the uint64_t is not the same as unsigned long long.  https://stackoverflow.com/questions/32198368/unsigned-long-long-conflict-with-uint64-t

the PRIu64 can correctly format uint64_t as lu or llu on different platforms

Reply:
another way is use llu, and convert uint64_t to (unsigned long long):

LOGE("SV_ID != SLOT_NUMBER: %d %llu", msg->sv_id(), (unsigned long long)data->n());